keyboards/ergodox_infinity/config.h

```diff
@@ -54,7 +54,20 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 /* key matrix size */
 #define MATRIX_ROWS 18
 #define MATRIX_COLS 5
-#define LOCAL_MATRIX_ROWS 9
+
+// For some reason, the rows are colums in the schematic, and vice versa
```

Comment:
This is probably because it's based on the ergodox (OG), which has it's matrix rotated 90 degrees.  So it's probably the same here. 

That said... if that's the case, it may be worth using the eager per row debounce algo for the board, if each row can only be used by one finger at a time.

Reply:
What I referred to here was the fact that the rows (as taken from the old custom matrix.c) are _named_ columns in the schematic, and vice versa. I left it in this state to avoid having to rewrite all the LAYOUT defines.

I've actually tried using the eager_pr debounce algo in the past (I did [some work](https://github.com/firetech/qmk_firmware/commit/a939debc0e0f4d43d34fa82810c5db34f4b3609a) last year to move over the Infinity to the common debounce handling, but ended up not submitting it), mostly for parity with the EZ I use at work, and actually got quite a lot of bouncing keys with that, so I went back to defer_g (which is the main reason I never submitted a PR, it didn't really add any value).

I'm currently using the defer_pk algo on my Infinity, for no particular reason other than having lots of RAM to spare. 😎